node/src/requests/debug.rs

```diff
@@ -159,8 +158,8 @@ impl<RequestType: Request, ChainRespondArgsType: ChainRespondArgs>
             .unwrap();
         }
         write!(&mut output, " elect:").unwrap();
-        for (i, participant) in leader_selection.iter().enumerate() {
-            if i == leader_selection.len() - 1 {
+        for participant in self.leader_selection_order.iter() {
+            if Some(*participant) == current_leader {
```

Comment:
These are not strictly equivalent, nevertheless it makes more sense to show the full list like now. Else we could add a `break` then it would be equivalent to what was there before.

Reply:
Sounds good.